tests/unittest/_torch/modeling/test_modeling_gemma2.py

```
@@ -0,0 +1,156 @@
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from copy import deepcopy
```

Flake8 (7.3.0)

[error] 'copy.deepcopy' imported but unused

(F401)